src/aux-server/aux-web/aux-projector/interaction/ClickOperation/BuilderEmptyClickOperation.ts

```diff
@@ -21,11 +21,24 @@ export class BuilderEmptyClickOperation extends BaseEmptyClickOperation {
     }
 
     protected _performClick(calc: FileCalculationContext): void {
+        appManager.simulationManager.primary.filePanel.isOpen = false;
+        appManager.simulationManager.primary.filePanel.restrictVisible(false);
+
+        this.RemoveSelected();
+
         this._game.gameView.$emit('onContextMenuHide');
+    }
 
-        appManager.simulationManager.primary.selection.clearSelection();
-        appManager.simulationManager.primary.filePanel.search = '';
+    async RemoveSelected() {
```

Comment:
@shaneYeti Can you use camel case naming for this function? To be consistent it should be `removeSelected()` instead of `RemoveSelected()`

Reply:
changed